src/editors/containers/VideoUploadEditor/index.jsx

```diff
@@ -120,7 +107,7 @@ const VideoUploadEditor = (
               onClick={handleCancel}
             />
           </div>
-          <VideoUploader onUpload={handleDrop} errorMessage={errorMessage} intl={intl} />
+          <VideoUploader onUpload={handleDrop} intl={intl} />
```

Comment:
Why are we passing `intl` here? We can just use `useIntl` in that component.

Reply:
useIntl is not available to us as pointed out in https://github.com/openedx/frontend-lib-content-components/blob/main/docs/decisions/0005-internal-editor-testability-decisions.md#additional-considrations

Reply:
I think it's available now, I've used it in my latest PR.